### PR TITLE
Use const instead of hard-code for kubeadm usages

### PR DESCRIPTION
**Pull request by xiangpengzhao**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
/cc @luxas 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by wackxu** [comment]
seems dup of https://github.com/kubernetes/kubernetes/pull/53929

**Comment by xiangpengzhao** [comment]
@wackxu cool! thanks for reminding :) I remove the dup commit and only keep the diff one here.

File: cmd/kubeadm/app/cmd/token.go [hunk 1]
```diff
@@ -221,7 +220,6 @@ func RunCreateToken(out io.Writer, client clientset.Interface, token string, tok
 		}
 	}
 
-	// TODO: Validate usages here so we don't allow something unsupported
```
Comment (luxas) [inline cmd/kubeadm/app/cmd/token.go]: Can you fix this TODO please?
With this I mean that there should exist a list of the supported usages (signing, authentication), and if you specify something else (like foo) it should error out.

File: cmd/kubeadm/app/cmd/token.go [hunk 2]
```diff
@@ -210,8 +208,9 @@ func RunCreateToken(out io.Writer, client clientset.Interface, token string, tok
 
 	// adding groups only makes sense for authentication
 	usagesSet := sets.NewString(usages...)
-	if len(extraGroups) > 0 && !usagesSet.Has("authentication") {
-		return fmt.Errorf("--groups cannot be specified unless --usages includes \"authentication\"")
+	usageAuthentication := strings.TrimPrefix(bootstrapapi.BootstrapTokenUsageAuthentication, bootstrapapi.BootstrapTokenUsagePrefix)
```
Comment (luxas) [inline cmd/kubeadm/app/cmd/token.go]: While you're here and mocking with these funcs, could you add some valuable unit tests for the functionality?
(might require splitting out some functionality into standalone funcs, but I don't mind)

**Review by luxas — COMMENTED** [review]
Great to get this sorted out.
Improvement suggestions you might wanna fix while you're here :)

File: cmd/kubeadm/app/cmd/token.go (hunk 2 above)
Comment (xiangpengzhao) [reply cmd/kubeadm/app/cmd/token.go]: will address later :)

File: cmd/kubeadm/app/cmd/token.go (hunk 1 above)
Comment (xiangpengzhao) [reply cmd/kubeadm/app/cmd/token.go]: actually, I fixed this in another commit of this PR, but @wackxu told me that it's dup of #53929, so I just remove that commit :)

File: cmd/kubeadm/app/cmd/token.go (hunk 2 above)
Comment (luxas) [reply cmd/kubeadm/app/cmd/token.go]: In this or an other PR you mean?

File: cmd/kubeadm/app/cmd/token.go (hunk 1 above)
Comment (luxas) [reply cmd/kubeadm/app/cmd/token.go]: then I wouldn't remove the comment here, instead let @wackxu do it in https://github.com/kubernetes/kubernetes/pull/53929

File: cmd/kubeadm/app/cmd/token.go (hunk 2 above)
Comment (xiangpengzhao) [reply cmd/kubeadm/app/cmd/token.go]: I'll be off work today soon at my local time. So I'd like to send another PR to address what you are expecting. WDYT? Feel free to merge this one first if you think it's good to go :)

File: cmd/kubeadm/app/cmd/token.go (hunk 1 above)
Comment (xiangpengzhao) [reply cmd/kubeadm/app/cmd/token.go]: OK :)

**Review by luxas — COMMENTED** [review]
/lgtm
/approve no-issue

**Review by luxas — APPROVED** [review]

**Pull request merged**